.github/workflows/ci-sharktank-and-shortfin.yml

```diff
@@ -70,31 +70,5 @@ jobs:
             iree-runtime \
             "numpy<2.0"
 
-      - name: Run llama test
-        run: ./build_tools/integration_tests/llama_export_compile_serve.sh
-
-  test_punet:
```

Comment:
Am I right to understand that llama tests makes the punet one unnecessary?

Reply:
Thread discussing the removal of this script:

https://github.com/nod-ai/SHARK-Platform/pull/373#discussion_r1823450607

Reply:
gotcha i missed that ty!